test/scripts/e2e.sh

```diff
@@ -123,7 +123,11 @@ if [ -z "$E2E_TEST_FILTER" ] || [ "$E2E_TEST_FILTER" == "SCRIPTS" ]; then
     python3 -m venv "${TEMPDIR}/ve"
     . "${TEMPDIR}/ve/bin/activate"
     "${TEMPDIR}/ve/bin/pip3" install --upgrade pip
-    "${TEMPDIR}/ve/bin/pip3" install --upgrade py-algorand-sdk cryptography
+    "${TEMPDIR}/ve/bin/pip3" install --upgrade cryptograpy
+    # get the latest tip of our python SDK main branch as opposed to what's on pypi:
+    "${TEMPDIR}/ve/bin/pip3" install --upgrade git+git://github.com/algorand/py-algorand-sdk@master
+    # enable remote debugging:
```

Comment:
Doesn't affect Circle tests. But of course, user beware. Make sure not to turn on the debugger when sending to Circle (...there might be a clever way to debug remotely to circle, but I don't recommend that)

Reply:
@jasonpaulos  @tsachiherman - pinning the python SDK version

Reply:
Not sure about pinning its version, the latest and greatest looks reasonable default. Otherwise do we potentially need to update it with each pysdk release?